app/src/main/java/ch/epfl/sdp/utilities/package-info.java

```diff
@@ -0,0 +1,4 @@
+/**
+ * Will deleted soon and the contents of Tools will move in other folders
```

Comment:
Where would you like to move them?

Reply:
I'd put the biometric stuff inside or in the package where we deal with biometrics and the others functions can go in the Coronagame class

Reply:
The SingleFragmentActivity can also go in the testing folder after some small changes in the main folder but Matteo's merge is already going to be very difficult. I don't want to make it worse